web/packages/core/package.json

```diff
@@ -10,26 +10,7 @@
         "dist/"
     ],
     "scripts": {
-        "//0": "# Setting ENABLE_WASM_EXTENSIONS=true causes a second module to be built as well,",
-        "//1": "# that utilizes WebAssembly extensions, instead of it just being a 'fake' - a copy",
-        "//2": "# of the 'vanilla' one.",
-        "//3": "# Unfortunately, we have to set `$RUSTFLAGS` here, instead of in `.cargo/config.toml`",
-        "//4": "# (for example), because it's not yet possible to specify them per-profile:",
-        "//5": "# https://github.com/rust-lang/cargo/issues/10271",
-        "//6": "# Enabling `build-std` would also be great, but it's not stable yet.",
-        "prebuild": "npm run build:wasm-vanilla && npm run build:wasm-extensions",
-        "build:wasm-vanilla": "cross-env OUT_NAME=ruffle_web CARGO_PROFILE=web-vanilla-wasm RUSTFLAGS=\"$RUSTFLAGS --cfg=web_sys_unstable_apis -Aunknown_lints\" npm run build:wasm",
-        "//7": "# Dispatches to either building the real, or copying the fake (stand-in),",
-        "//8": "# 'with-extensions' module.",
-        "build:wasm-extensions": "node -e \"process.exit(process.env.ENABLE_WASM_EXTENSIONS == 'true' ? 0 : 1)\" && npm run build:wasm-extensions-real || npm run build:wasm-extensions-fake",
-        "build:wasm-extensions-real": "echo \"Building module with WebAssembly extensions\" && cross-env OUT_NAME=ruffle_web-wasm_extensions CARGO_PROFILE=web-wasm-extensions RUSTFLAGS=\"$RUSTFLAGS --cfg=web_sys_unstable_apis -Aunknown_lints -C target-feature=+bulk-memory,+simd128,+nontrapping-fptoint,+sign-ext,+reference-types\" WASM_BINDGEN_FLAGS=\"--reference-types\" WASM_OPT_FLAGS=\"--enable-reference-types\" npm run build:wasm",
-        "build:wasm-extensions-fake": "echo \"Copying the vanilla module as stand-in\" && shx cp dist/ruffle_web_bg.wasm dist/ruffle_web-wasm_extensions_bg.wasm && shx cp dist/ruffle_web_bg.wasm.d.ts dist/ruffle_web-wasm_extensions_bg.wasm.d.ts && shx cp dist/ruffle_web.js dist/ruffle_web-wasm_extensions.js && shx cp dist/ruffle_web.d.ts dist/ruffle_web-wasm_extensions.d.ts",
```

Comment:
`shx` is no longer used, thus can be removed from `devDependencies`.

Reply:
Good catch! Removed